### PR TITLE
html & css popup

### PR DESCRIPTION
**Pull request by SergeyShytikov**
### HTML & CSS popup 

[Demo](https://sergeyshytikov.github.io/popupDemo/) |
[Code base](https://github.com/SergeyShytikov/popupDemo)

Please, review.

### CODE REVIEW

File: submissions/SergeyShytikov/html-css-practice-task/index.html [hunk 1]
```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
```
Comment (A-Ostrovnyy) [inline submissions/SergeyShytikov/html-css-practice-task/index.html:1]: Use [prettier](https://prettier.io/) for auto-formatting code.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
Some small issues need to fix)

File: submissions/SergeyShytikov/html-css-practice-task/slyle.css [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  color: #000;
+}
+
+/* header */
+.navigation {
+  width: 400px;
+  display: inline-flex;
+  justify-content: space-around;
+  align-items: center;
+  float: right;
```
[the remark this answers is not in the document]
Comment (SergeyShytikov) [reply submissions/SergeyShytikov/html-css-practice-task/slyle.css]: Thanks for your review! I deleted float and added a class for flexbox to a header.

**Comment by A-Ostrovnyy** [comment]
Well done!
After pushing your changes click `Re-request review` to inform your reviewer)

image

File: submissions/SergeyShytikov/html-css-practice-task/index.html (hunk 1 above)
Comment (SergeyShytikov) [reply submissions/SergeyShytikov/html-css-practice-task/index.html:1]: Prettier was installed to my vscode as an extension and I think works well. Maybe you can explain what I need to correct?
Screenshot 2021-09-14 at 23 13 12

**Pull request merged**